docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+# Bot Framework Labeling Guidelines (DRAFT)
+
+This article describes the common labels used in the Bot Framework repositories to tag issues and PRs.
+
+It also provides guidance on how to use them.
+
+## Label format
+
+In order to be able to parse the different labels in reports we use `:` to separate the categories and subcategories in a label. The general format of a label is:
+
+```bash
+Main category:Subcategory 1:Subcategory 2
+```
+
+There's no limit to the number of subcategories but normally 2 or 3 levels at most should be enough.
+
+## Label usage
+
+Some labels can be applied multiple times to the same issue and some other shouldn't. For example, an issue could be labeled as `Area:Adaptive` **and** `Area:Skills` but it should **only** have one priority label `P:0` or `P:1`.
```

Comment:
I suggest we made an exception for P0, P1 to not following the covention of a:b. P:0, P:1 looks very artificial and P0, P1 are well perceived terms to me.

Reply:
I agree with @boydc2014 and I separately asked @munozemilio about this.

That said, this is subjective (we the SDK Team are used to “P1”, etc) and if the largest OSS projects on GitHub use this format, then it is worth considering changing. 

Though perhaps it is objectively more complicated...

Reply:
It's more consistent to use the same format.  But... My big concern is the ripple effect of changes this could cause.  How many people will have to adjust their reporting for this?  Just Kamran for his daily reports, or a bunch of others?

Reply:
@stevengum the teams and companies i'ved work with, p0, p1 seems common, of course i only speak from my experience.  

@tracyboehrer maybe the reporting is easier than updating all exitsting labels into new format since those labels are not 1:1 mapping to old ones.

Reply:
Updated doc to use P_x_ instead of P_:x_